angularjs-portal-home/src/main/webapp/js/master-override.js

```diff
@@ -1,83 +1,81 @@
 define(['angular'], function(angular) {
-
-  /*Keep in sync with docs/markdown/configuration.md*/
+  /* Keep in sync with docs/markdown/configuration.md*/
 
     var config = angular.module('override', []);
     config
-        //see configuration.md for howto
+        // see configuration.md for howto
         .constant('OVERRIDE', {
-          'APP_FLAGS' : {
+          'APP_FLAGS': {
             'enableToggle': true,
-            'defaultView' : 'expanded',
-            'compact' : true,
-            'expanded' : true,
-            'isWeb' : true,
-            'defaultTheme' : 0,
-            'debug' : true,
-            'showUserSettingsPage' : true
+            'defaultView': 'expanded',
+            'compact': true,
+            'expanded': true,
+            'isWeb': true,
+            'defaultTheme': 0,
+            'debug': true,
+            'showUserSettingsPage': true,
```

Comment:
@apetro @thevoiceofzeke I'm not sure how far back you support IE.
Dangling commas will work fine in IE8+, it will cause a parser error in IE7 and below.